packages/react-components/src/molecules/SocialIcons.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { UserResponse } from '@asap-hub/model';
+import css from '@emotion/css';
+
+import { Link } from '../atoms';
+import {
+  linkedInIcon,
+  githubIcon,
+  twitterIcon,
+  researchGateIcon,
+  researcherIdIcon,
+  orcidSocialIcon,
+  googleScholarIcon,
+} from '../icons';
+import { perRem } from '../pixels';
+
+const socialContainerStyles = css({
+  display: 'grid',
+  columnGap: `${12 / perRem}em`,
+  gridAutoFlow: 'column',
+  alignItems: 'center',
+  lineHeight: 0,
+  paddingTop: `${15 / perRem}em`,
+  paddingBottom: `${17 / perRem}em`,
+});
+
+type SocialIconsProps = UserResponse['social'];
+
+const SocialIcons: React.FC<SocialIconsProps> = ({
```

Comment:
Probably enough to warrant its own story I'd say?

Reply:
I didn't think it was worth it as its only going to be used in the one place

Reply:
don't feel that strongly about it